bin/cron/analyze_hoc_activity

```diff
@@ -144,7 +146,7 @@ def main
     csedweek_entire_schools:  12754,
     csedweek_students:        4_875_091,
     csedweek_countries:       356,
```

Comment:
A separate thread is being started to see if these (`csedweek_*`) can be removed.